langserver/handler_go113.go

```diff
@@ -1,4 +1,4 @@
-// +build go1.13
+//go:build go1.13
```

Comment:
Go 1.13 does not know go:build

Reply:
Yes, I know, this was found by the `revive` linter.

Go `1.13` and `1.12` are currently unmaintained, do you still want to support them?

If so, I'll add a linter exception for those 2 files.

Reply:
I reverted the change and added an exception for `goimports`.